sdk/core/azure-core/test/ut/transport_adapter_curl.cpp

```diff
@@ -15,27 +17,6 @@ using testing::ValuesIn;
 
 namespace Azure { namespace Core { namespace Test {
 
-  /**********************   Define the parameters for the base test and a suffix  ***************/
-  namespace {
-    static Azure::Core::Http::TransportPolicyOptions GetTransportOptions()
-    {
-      Azure::Core::Http::TransportPolicyOptions options;
-      options.Transport = std::make_shared<Azure::Core::Http::CurlTransport>();
-      return options;
-    }
-
-    // When adding more than one parameter, this function should return a unique string.
-    // But since we are only using one parameter (the libcurl transport adapter) this is fine.
-    static std::string GetSuffix(const testing::TestParamInfo<TransportAdapter::ParamType>& info)
```

Comment:
Also fixes https://github.com/Azure/azure-sdk-for-cpp/issues/1087, correct?

Reply:
correct 👍